drivers/hmis/app/graphql/types/hmis_schema/ce_candidate.rb

```diff
@@ -34,6 +34,8 @@ def client_attributes
     end
 
     def enrollments # not for batch
+      return [] if source_clients.empty?
```

Comment:
This addresses problem 1 in my comment on the ticket https://github.com/open-path/Green-River/issues/8196#issuecomment-3282562603